Modules/Module_Processing/xASL_init_DataLoading.m

```diff
@@ -59,6 +59,12 @@
         else
             error('No rawdata or derivatives found to load subjects from, first convert DICOMs 2 BIDS');
         end
+
+        if isequal(x.opts.bProcess, [0 0 1])
```

Comment:
if somebody calls ExploreASL([0;0;1]) then this condition will not catch it. 
That's why I wrote bProcess(:) and [0;0;1] which is a universal solution